moviepy/video/io/ImageSequenceClip.py

```diff
@@ -78,14 +78,14 @@ def __init__(
 
         # check that all the images are of the same size
         if isinstance(sequence[0], str):
-            size = imread(sequence[0]).shape
+            size = imread(sequence[0], index=0)[0].shape
```

Comment:
```suggestion
            size = iio.improps(sequence[0], index=0).shape
```

Another advantage of using the full namespace is that you get [`improps`](https://imageio.readthedocs.io/en/stable/_autosummary/imageio.v3.improps.html#imageio.v3.improps) which gives you standardized metadata (including the shape of the resulting image) without decoding pixels (if possible).

This needs less memory and is _much_ faster since we only need to read a small header instead of a potentially large image.